packages/migrate/src/__tests__/MigrateDev.test.ts

```diff
@@ -651,7 +651,7 @@ describe('sqlite', () => {
 
     `)
     expect(ctx.mocked['console.log'].mock.calls).toEqual([])
-    expect(ctx.mocked['console.error'].mock.calls).toEqual([])
+    expect(ctx.mocked['console.error'].mock.calls).toMatchSnapshot()
```

Comment:
Putting this one into an inline snapshot would also be perfectly fine I guess for the tests, no need to have and look into an additional file. But of course not necessary to merge this PR now.